include/drivers/gps.h

```diff
@@ -196,6 +196,9 @@ typedef void (*gps_event_handler_t)(struct device *dev,
  * @typedef gps_start_t
  * @brief Callback API for starting GPS operation.
  *
+ * If gps is already running a call to this function will
+ * restart the gps.
+ *
```

Comment:
Please move it to the description of `gps_start()`. That's where users will look for information regarding the function behavior.

Reply:
done. That was actually where I intended it to be :)